jsBasics/homeWork5/task1.js

```diff
@@ -0,0 +1,22 @@
+//function declaration
+function calculationAreaOfRectangle(width, height){
```

Comment:
`calculateRectangleArea ` would be better name

function name should answer the question "what to do?"